src/cpp/benders/benders_core/include/BendersBase.h

```diff
@@ -29,25 +29,31 @@ class BendersBase {
   virtual void free() = 0;
   virtual void run() = 0;
   virtual void initialize_problems() = 0;
-  void init_data();
+  virtual void init_data();
```

Comment:
can we not test Benders classes without "virtualizing" function just for test?

Reply:
As discussed with @JasonMarechal , virtualization is not a problem even if used just for testing purposes.

Reply:
To add details to discussion log:
- "Virtual" add expansion point to class, however it is rarely an issue to have your class extended, only in specific cases.
- "Virtual" may effect performances, but it's often not an issue and not the first candidate for optimization.
- It's best to be able to test our classes than not being able to just to satisfy some assumptions.